### ISSUE TITLE
Drawing in the bootstrap 4 version?

### ISSUE THREAD

**Issue by spotUP** [kristopolous/BOOTSTRA.386#83]
Hello!

I tried the latest version yesterday but i couldn't get the fancy bbs style line drawing to work?
Is it supposed to work?

I had to use the 2.3 version for my project and .. well.. it's old. :D I failed to align an image the way  i want to. 

I am rebuilding a quite popular underground ascii community called asciiarena for bbs art guys.
 
Can I contact you guys on irc or via mail or something? It would be great to talk, i have some ideas for this theme that you might be interested in.

**Comment by kristopolous** [comment]
nope, doesn't exist yet. I started may 1 really and it's just me. You can just text me directly. factor 57015009877788133 and you'll see jenny's number and mine.

**Comment by spotUP** [comment]
i don't know how to do that factoring stuff. i am not a math guy. =)
when do you think you'll have v4 in a useable state?

**Comment by kristopolous** [comment]
Maybe end of month. You can do it at the command line at least in Linux. Use the factor program

**Comment by spotUP** [comment]
ok, i got two numbers.. you mean text like send a text via sms? :D that's.. oldskool i guess.

**Comment by kristopolous** [comment]
Yeah, just hit me up if you want. I'm ok with it.

**Comment by spotUP** [comment]
is this implemented now? =)

**Comment by spotUP** [comment]
Bump! It's so cool!

**Comment by spotUP** [comment]
Any news on this?
